app/uk/gov/hmrc/partnershipidentificationfrontend/models/PostCode.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.partnershipidentificationfrontend.models
+
+import uk.gov.hmrc.http.InternalServerException
+
+
+case class PostCode(stringValue: String) {
+
+  import PostCode._
+
+  val sanitisedPostCode: String = stringValue.toUpperCase.filterNot(_.isWhitespace) match {
+    case standardPostCodeFormat(outwardCode, inwardCode) => outwardCode + " " + inwardCode
+    case other => throw new InternalServerException(s"Invalid postcode format: $other") // should never happen as it is validated in the form
+  }
```

Comment:
Doesn't look like sanitisedPostcode is used anywhere. I think it was for the EACD call on claim enrolment.